### ISSUE TITLE
Benchmarking and documentation re: `super_partition()`

### ISSUE THREAD

**Issue by malcolmbarrett** [USCbiostats/partition#32]
@katelynqueen98, after #29 is merged, I think we should consider adding a few supporting elements to the package documentation:

* A link and explanation in the `partition()` documentation
* An example in the readme
* Maybe mention it in the intro vignette, as well, or even a short new vignette?

The other thing I was thinking about is if there should be a benchmark example like the ones in the current README

**Comment by katelynqueen98** [comment]
Yes, happy to work on it! In terms of vignettes, either way is fine with me. It's not very much new material, so adding it to the intro vignette seems reasonable, or we could do a larger example with a GEO dataset and create a new vignette.

**Comment by malcolmbarrett** [comment]
One option for a larger example is to do `usethis::use_article()`. This will create a vignette that only shows up in the website, meaning we don't have to deal with including a large dataset in the package binary (which CRAN doesn't like usually).

I'll leave it up to you, though. If you think there are things worth clarifying with a longer example, we should do that. If you think it's enough to show the basics in existing documentation because the pattern of use is the same, that's also reasonable. 

BTW for the benchmarking, I believe I did some trickery so that it doesn't run every time, meaning you can do a longer example. For this particular problem, you could also run the benchmarks in another repository and save the results to use in the README here. That assumes that it'd be helpful to show it in a larger example that would be impractical to include in this repo. Again, I'll trust your judgment on what is the best demonstration of `super_partition()`

**Comment by katelynqueen98** [comment]
I think the basics are sufficient. The only new thing in the `cluster_size` parameter, which can easily be explained in the existing documentation.

I'm not sure how useful the benchmarks will be, but I am including them in my dissertation, so I will run them regardless. If they add new information compared to whats already in the README, we'll add them, and if not, they will only exist for my dissertation.

**Comment by malcolmbarrett** [comment]
That makes sense. Re: benchmarks, just a suggestion. I'll leave it all to you. 

If and when you publish on this, make sure to add a citation to the documentation here!

**Comment by malcolmbarrett** [comment]
@katelynqueen98 I'm going to move ahead with a submission to CRAN without these additions for the sake of a release for your paper. Feel free to add them later, and I'll do a patch release
